docs: enforcement action override in ExpansionTemplate (#2300)
open-policy-agent/gatekeeper: website/docs/expansion.md

The `ExpansionTemplate` custom resource specifies:

- Which resource(s) should be expanded, specified by their GVK
- The GVK of the resultant resource
- Which field to use as the "source" for the resultant resource. The template
  source is a field on the parent resource which will be used as the base for
  expanding it before any mutators are applied. For example, in a case where a
  `Deployment` expands into a `Pod`, `spec.template` would typically be the
  source.

> - Optionally, an enforcement action override to use when validating resultant

Reviewer: ```suggestion
- Optionally, an enforcement action override can be used when validating resultant
```
